app/api/server/v1/users.js

```diff
@@ -378,6 +380,7 @@ API.v1.addRoute('users.updateOwnBasicInfo', { authRequired: true }, {
 			email: this.bodyParams.data.email,
 			realname: this.bodyParams.data.name,
 			username: this.bodyParams.data.username,
+			statusMessage: this.bodyParams.data.statusMessage,
```

Comment:
would be really cool if like setAvatar this was a rest call.  Imagining the ability to set status from music players, and many other much more practical integrations.

Reply:
Like my smart toaster! "toast is done".

I'll get this coded up when I wake up unless someone else wants to give it a whack before then.

Reply:
OK, pushed. @geekgonecrazy please take a look at the changes and let me know if I should change it.